docs/resources/metal_vrf_bgp_dynamic_neighbor.md

```diff
@@ -0,0 +1,65 @@
+---
+subcategory: "Metal"
+---
+
+# equinix_metal_vrf_bgp_dynamic_neighbor (Resource)
+
+This resource manages BGP dynamic neighbor ranges for an Equinix Metal VRF, but with markdown
```

Comment:
"but with markdown"?

Reply:
Related to [this comment thread](https://github.com/equinix/terraform-provider-equinix/pull/746#discussion_r1700649984)